.run/watch-scripts.run.xml

```diff
@@ -1,11 +0,0 @@
-<component name="ProjectRunConfigurationManager">
```

Comment:
@elchugreeva Was this file changed by accident or is it part of this diff on purpose?